usaspending_api/api_docs/markdown/using_the_api.md

```diff
@@ -11,25 +11,24 @@
   </ul>-->
```

Comment:
@russellbodine this file still has a lot of v1 documentation present. is there a reason this is still here?

Reply:
@nmonga91 No reason besides that its necessary information for a user who has to use v1 endpoints.